modules/developers-guide/pages/default-wallet.adoc

```diff
@@ -17,54 +17,206 @@ Because only canisters require and consume cycles—to perform operations and to
 For the purposes of local development, the {sdk-short-name} automatically creates a default cycles wallet for you in every project and most of the operations performed using the cycles wallet happen behind the scenes.
 For example, the cycles wallet acts on your behalf to register canister identifiers and deploy canisters on the local network.
 
-In a production environment, however, you often need to explicitly register and transfer cycles to new canisters, specify the principals that can act as custodians, and manage the principals with ownership rights. 
-You can perform these types of tasks using the methods in the default cycles wallet.
+In a production environment, however, you often need to explicitly register and transfer cycles to new canisters, specify the principals that can act as custodians, and manage the principals with ownership rights.
+You can perform some of these tasks using the default cycles wallet application running in a web browser. 
+Depending on the specific action you want to take, you can also perform these cycle and canister management tasks by running `+dfx wallet+` commands in a terminal or by calling methods in the default cycles wallet canister directly.
 
 You should keep in mind, however, that calls to the cycles wallet canister are executed using the cycles wallet identifier associated with the currently-selected user identity.
 Depending on your currently-selected identity and whether the principal associated with that identity has been added as a controller or a custodian for a wallet, you might see different results or be denied access to a specific method.
 
-[[wallet-create-wallets]]
-== Create a new cycles wallet
+To check the identity you are currently using, run the following command:
 
-Use the `wallet_create_wallet` method to create a new cycles wallet canister with an initial cycle balance and, optionally, with a specific principal as its controller.
-If you don't specify a controlling principal, the cycles wallet you use to create the new wallet will be the new wallet's controller.
+[source,bash]
+----
+dfx identity whoami
+----
 
-For example, you can run a command similar to the following to create a new wallet and assign a principal as a controller:
+== Controller and custodian roles
+
+A user principal or canister identifier can be assigned to a *controller* or *custodian* role.
```

Comment:
Note that granting the controller role also grants the custodian role. If you deauthorize an identity as a custodian, it also removes it as a controller